buildspec.yml

```diff
@@ -2,32 +2,32 @@ version: 0.1
 
 environment_variables:
   plaintext:
-    DATE_FORMAT: '%F_%H:%M:%S'
+    APP_NAME: 'suitec-preview-service'
```

Comment:
Fyi, this is a non-essential use of `environment_variables` but I want to verify that these env vars are available in phases below.